scripts/build-image.sh

```diff
@@ -48,5 +48,6 @@ npm run env:stop-base
 echo "Copying meta files to the JS library"
 rm -rfv "$JS_LIB_CONTRACTS_DIR"/*
 cp -a "$ROOT_PATH/artifacts/contracts/." "$JS_LIB_CONTRACTS_DIR"
+cp -a "$ROOT_PATH/artifacts/@ensdomains/ens-contracts/contracts/." "$JS_LIB_CONTRACTS_DIR"
```

Comment:
we don't need all ens-contract artifacts as we just use `ENSRegistry`, but that is defined in the `contracts` already.